src/clients/node/docs.zig

```diff
@@ -0,0 +1,310 @@
+const Docs = @import("../docs_types.zig").Docs;
+
+pub const NodeDocs = Docs{
+    .readme = "node/README.md",
+    .markdown_name = "javascript",
+    .extension = "js",
+    .test_linux_docker_image = "node:18",
+    .name = "tigerbeetle-node",
+    .description = 
+    \\The TigerBeetle client for Node.js.
+    ,
+    .prerequisites = 
+    \\* NodeJS >= `14`. _(If the correct version is not installed, an installation error will occur)_
+    \\
+    \\> Your operating system should be Linux (kernel >= v5.6) or macOS.
+    \\> Windows support is not yet available.
+    ,
+
+    .install_sample_file = 
+    \\const { createClient } = require("tigerbeetle-node");
+    \\console.log("Import ok!");
+    ,
+
+    .install_sample_file_build_commands = "npm install typescript @types/node && npx tsc --allowJs --noEmit test.js",
```

Comment:
Phil: Create a new repo that publishes docker images (weekly?) that includes these dependencies so we can use instead of npm installing every time.

Reply:
Done

Reply:
Wasn't much faster with this than without